lib/forest_liana.rb

```diff
@@ -37,9 +37,23 @@ module UserSpace
   self.names_overriden = {}
   self.meta = {}
 
+  @config_dirs = []
+
   # TODO: Remove once lianas prior to 2.0.0 are not supported anymore.
   self.names_old_overriden = {}
 
+  def self.config_dirs=(dirs)
+    @config_dirs = (dirs || []).map { |dir| Rails.root.join(dir) }
+  end
+
+  def self.config_dirs
+    [
+      Rails.root.join('lib/forest/**/*.rb'),
+      Rails.root.join('lib/forest_liana/**/*.rb'),
+      *@config_dirs
+    ]
+  end
```

Comment:
Everything looks great.

For consistency with other lianas/stacks, could we consider having a single repository instead of an array of repositories?
I also think it would be a better idea as it forces developers to chose a single repository to write their Forest Admin customization.
What do you think?

Reply:
Resolved in 5005b94